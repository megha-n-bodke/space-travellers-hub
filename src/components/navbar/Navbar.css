.header {
  width: 100%;
  height: 100px;
  background-color: rgb(252, 252, 252);
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
}

.nav-des {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  margin-right: 40px;
}

.div-logo {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
}

.logo {
  width: 50px;
  height: 50px;
  margin-left: 60px;
  padding-top: 8px;
}

.nav-link {
  margin-left: 20px;
  margin-right: 20px;
  font-size: 18px;
  color: rgb(0, 13, 255);
  font-weight: normal;
  font-family: Poppins, sans-serif;
  text-decoration: none;
}

.me {
  background-color: palevioletred;
}

hr {
  border: solid 1px rgb(0, 0, 0);
  height: 20px;
  background-color: black;
}
