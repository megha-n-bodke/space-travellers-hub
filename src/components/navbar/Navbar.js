import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../../images/logo.png';
import './Navbar.css';

const Navbar = () => (
  <div className="header">
    <div className="div-logo">
      <img className="logo" src={logo} alt="user" />
      <h1>Space Travelers&apos; Hub</h1>
    </div>
    <nav className="nav-des">
      <NavLink className="nav-link" activeclassname="active" to="/">
        Rockets
      </NavLink>
      <NavLink className="nav-link" activeclassname="active" to="/missions">
        Missions
      </NavLink>
      <hr />
      <NavLink className="nav-link" activeclassname="active" to="/my-profile">
        My Profile
      </NavLink>
    </nav>
  </div>
);

export default Navbar;
