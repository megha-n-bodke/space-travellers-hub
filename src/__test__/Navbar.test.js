import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/navbar/Navbar';

it('Check any changes to the component', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
