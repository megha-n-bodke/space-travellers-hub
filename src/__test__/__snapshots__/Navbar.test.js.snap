// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check any changes to the component 1`] = `
<div
  className="header"
>
  <div
    className="div-logo"
  >
    <img
      alt="user"
      className="logo"
      src="logo.png"
    />
    <h1>
      Space Travelers' Hub
    </h1>
  </div>
  <nav
    className="nav-des"
  >
    <a
      activeclassname="active"
      aria-current="page"
      className="nav-link active"
      href="/"
      onClick={[Function]}
    >
      Rockets
    </a>
    <a
      activeclassname="active"
      className="nav-link"
      href="/missions"
      onClick={[Function]}
    >
      Missions
    </a>
    <hr />
    <a
      activeclassname="active"
      className="nav-link"
      href="/my-profile"
      onClick={[Function]}
    >
      My Profile
    </a>
  </nav>
</div>
`;
